WordPress/src/main/res/layout/site_settings_timezone_bottom_sheet_list_item.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/item_time_zone"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?attr/selectableItemBackground"
+    android:paddingTop="@dimen/margin_extra_large"
+    android:paddingBottom="@dimen/margin_extra_large">
+
+    <TextView
+        android:id="@+id/time_zone"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingBottom="@dimen/margin_small"
+        android:textAppearance="?attr/textAppearanceSubtitle1"
+        android:maxLines="1"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Abidjan" />
+
+    <TextView
+        android:id="@+id/zone_offset"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:textAppearance="?attr/textAppearanceBody2"
+        android:maxLines="1"
+        android:textColor="@color/zs_request_list_light_text_color"
```

Comment:
We probably would want to use something like `?attr/wpColorOnSurfaceMedium` here, so it will be readable in dark mode. I think `zs_request_list_light_text_color` comes from one of the 3rd party libraries.